src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -63,6 +63,11 @@ internal static class LogRecordExtensions
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
+        {
+            return logRecord.ToOtlpLog(SdkConfiguration.Instance.AttributeCountLimit ?? int.MaxValue, SdkConfiguration.Instance.AttributeValueLengthLimit);
+        }
+
+        internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord, int attributeCountLimit, int? attributeValueLengthLimit)
```

Comment:
Add `[MethodImpl(MethodImplOptions.AggressiveInlining)]` for this method.

Reply:
Done / additional overload is no longer required because of the changed SdkLimitOptions class.